.gitignore

```diff
@@ -170,3 +170,5 @@ results
 /.vscode/
 /img/
 *.png
+/test.py
+/result.xlsx
```

Comment:
To jest dobry pomysł żeby wyniki były w gitgnorze, ale może warto byłoby wrzucić przykładowe wyniki do `results`? Tak oprócz tego

Reply:
Dobrze dodam je dzisiaj